[ConfigPrinter] Undefined index: serviceKeywords

Hi,

after running `vendor/bin/config-transformer switch-format any-file.yaml` with the current version of config-transformer and php-config-printer (9.0.45), the following error is thrown:
```
PHP Notice:  Undefined index: serviceKeywords in /[redacted]/vendor/symplify/php-config-printer/src/Yaml/CheckerServiceParametersShifter.php on line 230

Notice: Undefined index: serviceKeywords in /[redacted]/vendor/symplify/php-config-printer/src/Yaml/CheckerServiceParametersShifter.php on line 230

                                                                                                                        
 [ERROR] in_array() expects parameter 2 to be array, null given 
```

The issue is caused by a [change](https://github.com/symfony/dependency-injection/compare/v5.2.1...v5.2.2#diff-55d41a75608dadf9346277d2e75c2c7e0f82944e139fd2c53ce313460eae9bb9R42) in `symfony/dependency-injection` 5.2.2 (released yesterday).

The reflected `private static $serviceKeywords` was replaced by `private const SERVICE_KEYWORDS`.

--- a/packages/php-config-printer/src/Yaml/CheckerServiceParametersShifter.php
+++ b/packages/php-config-printer/src/Yaml/CheckerServiceParametersShifter.php
@@ -48,6 +48,16 @@ final class CheckerServiceParametersShifter
     private const SERVICES_KEY = 'services';
 
     /**
+     * @var string
+     */
+    private const SERVICE_KEYWORDS_KEY_STATIC = 'serviceKeywords';
+
+    /**
+     * @var string
+     */
+    private const SERVICE_KEYWORDS_KEY_CONST = 'SERVICE_KEYWORDS';
+
+    /**
      * @var string[]
      */
     private $serviceKeywords = [];
@@ -224,11 +234,19 @@ final class CheckerServiceParametersShifter
     private function initializeServiceKeywords(): void
     {
         $reflectionClass = new ReflectionClass(YamlFileLoader::class);
+        /** @var array<string, mixed> $constants */
+        $constants = $reflectionClass->getConstants();
+        if (array_key_exists(self::SERVICE_KEYWORDS_KEY_CONST, $constants)) {
+            /** @var string[] $serviceKeywordsProperty */
+            $serviceKeywordsProperty = $constants[self::SERVICE_KEYWORDS_KEY_CONST];
+            $this->serviceKeywords = $serviceKeywordsProperty;
+            return;
+        }
+
         /** @var array<string, mixed> $staticProperties */
         $staticProperties = $reflectionClass->getStaticProperties();
         /** @var string[] $serviceKeywordsProperty */
-        $serviceKeywordsProperty = $staticProperties['serviceKeywords'];
-
+        $serviceKeywordsProperty = $staticProperties[self::SERVICE_KEYWORDS_KEY_STATIC];
         $this->serviceKeywords = $serviceKeywordsProperty;
     }
 }

